src/Components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -65,7 +68,7 @@ export const OverflowMenu: React.FC<IOverflowMenuProps> = ({
                 ariaLabel={ariaLabel || t('more')}
                 className={className}
                 componentRef={menuRef}
-                data-is-focusable={false}
+                data-is-focusable={isFocusable ?? false}
```

Comment:
the `?? false` seems redundant here

Reply:
it just makes sure that it gets a value I guess?

Reply:
I'm betting that if you don't pass a value, it defaults to true in this case. I could do !!isFocusable though.

Reply:
Done.